.github/workflows/ci-participant-portal.yml

```diff
@@ -113,11 +113,26 @@ jobs:
         with:
           useLockFile: false
           working-directory: ./participant
+      - name: Collect Docker Logs
+        uses: jwalton/gh-docker-logs@v2.2.1
+        with:
+          images: 'participant-app-e2e'
 
       - name: 🐳 Run Playwright in Docker Compose
         run: npm run e2e
         working-directory: participant
-
+      - name: Collect Docker Logs
+        if: failure()
+        uses: jwalton/gh-docker-logs@v2.2.1
+        with:
+          images: 'participant-app-e2e,postgres'
+          dest: './logs'
```

Comment:
It doesn't look like this is pulling down the logs for the app image? https://github.com/navapbc/wic-participant-recertification-portal/actions/runs/4746337014 looks like it only has logs for the db container